data/en/2_0001.json

```diff
@@ -154,7 +154,28 @@
                 },
                 {
                     "id": "number-of-employees-split-block",
+                    "title": "Quarterly Business Survey",
+                    "type": "Question",
                     "questions": [{
+                        "calculated": {
+                            "calculated_type": "sum",
+                            "answer_id": "number-of-employees-total",
+                            "answers_to_calculate": [
+                                "number-of-employees-male-more-30-hours",
+                                "number-of-employees-male-less-30-hours",
+                                "number-of-employees-female-more-30-hours",
+                                "number-of-employees-female-less-30-hours"
+                            ],
+                            "conditions": [
+                                "less than",
+                                "equals"
+                            ]
+                        },
+                        "validation": {
+                            "messages": {
+                                "TOTAL_SUM_NOT_LESS_THAN_OR_EQUALS": "Enter answers that add up to or are less than the Total number of employees, %(total)s"
+                            }
```

Comment:
I think we just want to use the standard error messages @charlotte-hirst can you confirm?

Reply:
yes standard message without piping of answer label please

Reply:
@shahi645 @charlotte-hirst we need to make sure that offline conversations are updated in the specs